compliance/cis_k8s/rules/cis_4_2_13/rule.rego

```diff
@@ -0,0 +1,50 @@
+package compliance.cis_k8s.rules.cis_4_2_13
+
+import data.compliance.cis_k8s
+import data.compliance.lib.common
+import data.compliance.lib.data_adapter
+
+default rule_evaluation = true
+
+process_args := cis_k8s.data_adapter.process_args
+
+supported_ciphers = [
+	"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256", "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+	"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305", "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+	"TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305", "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+	"TLS_RSA_WITH_AES_256_GCM_SHA384", "TLS_RSA_WITH_AES_128_GCM_SHA256",
+]
+
+# Ensure that the Kubelet only makes use of Strong Cryptographic Ciphers
+rule_evaluation = false {
+	ciphers := split(process_args["--tls-cipher-suites"], ",")
+	cipher := ciphers[_]
+	not is_supported_cipher(cipher)
+}
```

Comment:
nit: I think we could have use `some` [keyword](https://www.openpolicyagent.org/docs/latest/policy-language/#some-keyword) here